src/aap_eda/api/serializers/credential_type.py

```diff
@@ -53,10 +53,36 @@ class CredentialTypeCreateSerializer(serializers.ModelSerializer):
     )
 
     def validate(self, data):
-        if data.get("injectors") and data.get("inputs"):
-            errors = validate_injectors(
-                data.get("inputs"), data.get("injectors")
-            )
+        injectors = data.get("injectors")
```

Comment:
@hsong-rh In the CredentialTypeCreateSerializer inputs is marked as required=True should we have the same for injectors.
https://github.com/ansible/eda-server/blob/25deac499e99af1f892d9b7927e76dd7a6b84363/src/aap_eda/api/serializers/credential_type.py#L43

Reply:
@mkanoor for updating credential types, `inputs` or `injectors` should mark as required=False. we need to define a separate updating serializer.